package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-effects-view",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Not so many changes there. 0.4.0 fine by me. 👍

On Wednesday, June 22, 2016, Dmitri Voronianski <notifications@github.com>
wrote:

> In package.json
> <https://github.com/voronianski/react-native-effects-view/pull/11#discussion_r68157094>
> :
>
> > @@ -1,6 +1,6 @@
> >  {
> >    "name": "react-native-effects-view",
> > -  "version": "0.3.0",
> > +  "version": "0.3.1",
>
> @eugenehp <https://github.com/eugenehp> maybe 0.4.0?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/voronianski/react-native-effects-view/pull/11/files/43c85e2f99b4284367b23888cd3d919126a7e912#r68157094>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABxW76tvk-WNkQ1vDhsPIDIDSqB3IqKTks5qOc1ggaJpZM4I8Txx>
> .
>